tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java

```diff
@@ -95,12 +103,48 @@ public Tablet(String deviceId, List<MeasurementSchema> schemas, int maxRowNumber
     this.deviceId = deviceId;
     this.schemas = schemas;
     this.maxRowNumber = maxRowNumber;
+    mschemawithindex=new HashMap<>();
+
+    for(int i=0;i<schemas.size();i++){
+      mschemawithindex.put(schemas.get(i),i);
+    }
 
     createColumns();
 
     reset();
   }
 
+  public void addTimestamp(int rowIndex, long timestamp) {
+    timestamps[rowIndex] = timestamp;
+  }
+
+  public void addValue(MeasurementSchema measurementSchema, int rowIndex, Object value) {
```

Comment:
```suggestion
  public void addValue(String measurement, int rowIndex, Object value) {
```

Using measurement id may be more friendly to users.

Reply:
copy that,thank you